src/components/program-progress/ProgramProgressSidebar.jsx

```diff
@@ -69,10 +71,10 @@ const ProgramProgressSideBar = () => {
           <ul className="certificate-list">
             {courseCertificates.map((certificate) => (
               <li key={uuidv4()} data-testid="certificate-item" className="certificate">
-                <a className="image-link" href={certificate.url} aria-hidden="true" tabIndex="-1">
+                <a className="image-link" href={`${LMS_BASE_URL + certificate.url}`} aria-hidden="true" tabIndex="-1">
```

Comment:
nit: Using JS template string here vs. the old-school JS string concatenation would be preferred.

`${LMS_BASE_URL}${certificate.url}`

Reply:
nit x2: Given its used twice, maybe update the `map` function to create a `const certificateUrl` variable that can be re-used more readily.